src/LoginServer/Util/Configuration/Files/Login.cs

```diff
@@ -18,13 +18,30 @@ public class LoginConfFile : ConfFile
 		/// </summary>
 		public Location StartLocation { get; protected set; }
 
+		/// <summary>
+		/// When enabled, client integrity checks are applied.
+		/// </summary>
+		public bool VerifyIpf { get; private set; }
+
+		/// <summary>
+		/// Checksum to prevent clients using modified or out-dated IPF files.
+		/// </summary>
+		public string IpfChecksum { get; private set; }
+
 		public void Load()
 		{
 			this.Require("system/conf/login.conf");
 
+			this.IpfChecksum = this.GetString("ipf_checksum");
 			this.StartLocation = this.LoadStartLocation();
 		}
 
+		private void LoadIpfSettings()
```

Comment:
You're not calling this method, so `VerifyIpf` wouldn't be set. You're also getting `IpfChecksum` twice. You can put both directly into `Load`, like in the other conf files, `LoadStartLocation` is only a thing because it's more involved than a simple get call. You might also want to use default values in case some user deletes the options or conf files, and for `VerifyIpf` you can use `GetBool` instead of `GetString`. Saves you the mannual check and it supports "yes" instead of "true" (so fancy |D).